docs/docs/contextual-conversations.mdx

```diff
@@ -1,120 +1,182 @@
 ---
 id: contextual-conversations
-sidebar_label: Complex Contextual Conversations
-title: Complex Contextual Conversations
+sidebar_label: Contextual Conversations
+title: Contextual Conversations
 ---
 
-Not every user goal you define will fall under the category of business logic. For the
-other cases you will need to use stories and context to help the user achieve their goal.
-
-If we take the example of the “getting started” skill from Sara, we want to give them
-different information based on whether they've built an AI assistant before and are
-migrating from a different tool etc. This can be done quite simply with stories and
-the concept of [max history](./policies.mdx#max-history).
-
-```md
- ## new to rasa + built a bot before
- * how_to_get_started
-   - utter_getstarted
-   - utter_first_bot_with_rasa
- * affirm
-   - action_set_onboarding
-   - slot{"onboarding": true}
-   - utter_built_bot_before
- * affirm
-   - utter_ask_migration
- * deny
-   - utter_explain_rasa_components
-   - utter_rasa_components_details
-   - utter_ask_explain_nlucorex
- * affirm
-   - utter_explain_nlu
-   - utter_explain_core
-   - utter_explain_x
-   - utter_direct_to_step2
-
- ## not new to rasa + core
- * how_to_get_started
-   - utter_getstarted
-   - utter_first_bot_with_rasa
- * deny
-   - action_set_onboarding
-   - slot{"onboarding": false}
-   - utter_ask_which_product
- * how_to_get_started{"product": "core"}
-   - utter_explain_core
-   - utter_anything_else
+To create a context-aware conversational assistant, you will need to define
+how your assistant uses the conversation history to affect the next response.
```

Comment:
why do people need a context-aware assistant? We should describe use cases for this that are relevant. slightly similar to what we had before maybe. And can we add a short summary of the page under the `abstract:` key above?

Reply:
could also say that conversations can go in different ways based on things users say, or information they provide etc